service/Authentication.ts

```diff
@@ -128,7 +128,9 @@ export default class AuthenticationService {
   async register(userData) {
     const {
       username,
-      password
+      password,
+      details,
```

Comment:
Not sure if the `details` and `clientConfig` can be added automatically from the model here